### DISCUSSION TITLE
Github process information button not working.

### DISCUSSION THREAD

**Discussion by rizmyabdulla** [community/community#88401] [GitHub Education] [answered]
### Hi everyone,

Recently, I submitted the details of my campus to get a GitHub Developer Pack. However, after filling out all the information and clicking the "Process Information" button, the page took a long time to load and did not redirect to another page.

To investigate the issue, I opened the developer tools console and found some error messages. I have attached a screenshot of the error for reference.

![Screenshot 2024-01-16 154315](https://github.com/community/community/assets/89332851/09e5b002-745a-4159-8653-602363adb8ae)

Has anyone encountered a similar issue when applying for a GitHub Developer Pack? Any suggestions on how to resolve this problem would be greatly appreciated.

Thank you in advance for your help!

**Comment by KPaul404** [comment, marked as answer]
JavaScript error has occurred on the page preventing it from operating correctly. The specific error is a "TypeError" indicating an undefined property called "redirect_url" that the code is trying to access. This usually happens when the code attempts to use an object or property that does not exist.

Please try the following troubleshooting tips:

First, refresh the page or clear your browser's cache and cookies. This resets things and catches temporary glitches.

Next, check for browser extensions that could be interfering with the JavaScript code and disable them.

Also, Tryiing a different browser can also help rule out compatibility issues.

**Comment by KPaul404** [comment]
Also, sometimes its just your connection. Check your internet connection is working properly and that network settings are not blocking resources from loading.

**Reply by rizmyabdulla** [reply]
I tried and it didn't work 🥲

**Reply by KPaul404** [reply]
Oh, well I had a similar issue a while back but that resolved mine. You can also try what I had suggested on the first comment 😊

**Reply by rizmyabdulla** [reply]
Thanks for your suggestion 💖

**Comment by afeuerstein** [comment]
I also have the same problem. Checking the network tab shows the error:
![image](https://github.com/community/community/assets/32029275/9e7e7b7a-a3b1-45e7-bfa7-4c9560e03222)
Next to the upload button the normally accepted file sizes are listed as 100KB-10MB. The file I am trying to upload is 2.5MB big, so it should work.
Very probable that this is an issue on GitHub's side.
Also very unprofessional that they redirect you to the community forum when you try to open a support request about a bug.

**Comment by afeuerstein** [comment]
Reducing the size of the image worked for me. I got it down to 250KB.
Anything under 1MB should be OK, since that is the default payload size for nginx.

**Comment by afeuerstein** [comment]
> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted.
> - The image you used appears to be edited. Manipulated images may not be used. Please try a different image.

Duh, I shrunk the image so that I could apply.
Maybe wait this out until GitHub fixes this. I will try to contact support...

**Reply by mishmanners** [reply]
Please also ensure you try out these things: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514

**Comment by jethiya-coder** [comment]
I encountered the same . An easy fix for me : I connected my pixel 6a to my pc with usb debugging enabled and set my device as a webcam. Nothing extra requited , i clicked the photo through the phone cam

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules
